integration_tests/test_asset_group_sightings.py

```diff
@@ -23,21 +23,21 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
         {'commonNames': ['Example'], 'scientificName': 'Exempli gratia'},
     )
     tx_id = response.json()['response']['value'][-1]['id']
-    occ_test_cfd = utils.create_custom_field(
+    sighting_test_cfd = utils.create_custom_field(
```

Comment:
As agreed in discussion with @naknomum on last PR. The references to occurance are still required but minimise hem as much as possible